src/child.cpp

```diff
@@ -103,6 +103,14 @@ static void validate_cmd(std::vector<std::string>& cmd)
     case 1:
       cmd[0] = paths.front().c_str();
       break;
+    case 2:
+      // /bin maybe is a symbolic link to /usr/bin (/bin -> /usr/bin)
+      if (get_absolute_path(paths[0]) == get_absolute_path(paths[1]))
```

Comment:
Use an existing helper function `abs_path`:
```suggestion
      if (abs_path(paths[0]) == abs_path(paths[1]))
```

Reply:
Sorry, Just fix it, thank you.